### ISSUE TITLE
License laundering

### ISSUE THREAD

**Issue by aidan-fitz** [pages-themes/cayman#6]
The [original theme](https://github.com/jasonlong/cayman-theme) is licensed under CC-BY 4.0. You don't have the authority to release it into the public domain. Please change it back. :disappointed: 

Also, without attribution to the original author (Jason Long), this is copyright infringement.

**Comment by benbalter** [comment]
@aidan-fitz thanks for the concern, but your accusations, while I'm sure well intentioned are unfounded. 

@jasonlong is a GitHub employee and created the theme under [the work for hire doctrine](https://en.wikipedia.org/wiki/Work_for_hire). GitHub has the right to release the work under any license it chooses, including a public domain dedication as we did here (a less restrictive license that the one you're recommending).

We chose CC-0 over CC-BY to allow end-users to modify the theme as they saw fit, without needing to leave the author's name on it (as we did with all other themes we released at the time). In the past, author attribution was a sign of frustration and confusion as publishers and visitors would email the theme author for support.

As for attribution, even though not legally required, it's important that we give credit where credit is due, and [did so in the theme metadata](https://github.com/pages-themes/cayman/blob/master/jekyll-theme-cayman.gemspec#L7).
